CHANGELOG.md

```diff
@@ -2,6 +2,26 @@
 
 ### Bugfixes
 
+* Lorem ipsum.
+
+### Breaking changes
+
+* Lorem ipsum.
+
+### Enhancements
+
+* Lorem ipsum.
+
+### Internals
+
+* Implemented inter-process CondVars on Windows (Win32 + UWP). They should be fair and
+  robust.
+
+----------------------------------------------
+# 2.6.2 Release notes
+
+### Bugfixes
+
```

Comment:
More generally; this PR's changes to the `CHANGELOG.md` should only be adding a single entry, not changing any sections etc.